tests/unit/src/test/scala/tests/cli/CliSuite.scala

```diff
@@ -187,4 +189,24 @@ class CliSuite extends BaseCliSuite {
       |""".stripMargin
   )
 
+  checkCli(
+    "single-file",
+    """
+      |/index.md
```

Comment:
Can we add an additional sibling markdown file to see what it does not get processed?

Reply:
Yup, good idea.